SUSE/rmt
# Fixes for ltss

## Description

The instance data for instance verification is inside `params` but inside `hwinfo` within `params`

## How to test 

Start a SLES PAYG instance, registered run `registercloudguest -r <regcode>`
Instance verification should succeed

## Change Type

*Please select the correct option.*


## Checklist

*Please check off each item if the requirement is met.*


## Review

Please check out our [review guidelines](https://github.com/SUSE/scc-docs/blob/master/team/workflow/code_review.md) 
and get in touch with the author to get a shared understanding of the change.
